src/matchers.ts

```diff
@@ -84,6 +84,9 @@ export const hiddenDiv: Matcher = predicates.AND(
     predicates.hasAttr('hidden'),
     predicates.hasAttr('by-polymer-bundler'));
 export const inHiddenDiv: Matcher = predicates.parentMatches(hiddenDiv);
+export const baseTargetAppliesTo: Matcher = predicates.AND(
```

Comment:
As this is only used one place, recommend placing it in that file. More legible, easier to evaluate that it does what's expected.